lib/newman_conway.js

```diff
@@ -1,11 +1,39 @@
+// Newman-Conway sequence: P(n) = P( P(n - 1)) + P(n - P(n - 1)), with intial conditions P(1)=1 and P(2)=1
+// Time Complexity: O(n) - there is a for loop in newmanConway that will run n number of times. The function p that it calls on has a time complexity of O(1) because it directly looks up the values in the array it's adding to.
+// Space Complexity: O(n) - both the newmanConway and p functions add n elements to their data structures.
 
-// Time Complexity:
-// Space Complexity:
+newmanConway = (num) => {
```

Comment:
⚠️ , this works, but because you're not using your memo `pValues`, your recursion is making this O(3^n) .

Instead you should loop from 1 to num calculating the newmanConway value and storing it in your "memo." for lookup later.